src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -120,7 +120,7 @@
             <tag name="sylius.api.command_data_transformer" />
         </service>
 
-        <service id="sylius.api.data_transformer.add_product_review_input_data_transformer" class="Sylius\Bundle\ApiBundle\DataTransformer\LoggedInShopUserEmailAwareCommandDataTransformer">
+        <service id="sylius.api.data_transformer.logged_in_shop_user_email_aware_command_data_transformer" class="Sylius\Bundle\ApiBundle\DataTransformer\LoggedInShopUserEmailAwareCommandDataTransformer">
```

Comment:
There is duplicated `data_transformer` in service id
```suggestion
        <service id="sylius.api.data_transformer.logged_in_shop_user_email_aware_command" class="Sylius\Bundle\ApiBundle\DataTransformer\LoggedInShopUserEmailAwareCommandDataTransformer">
```
or
```suggestion
        <service id="sylius.api.logged_in_shop_user_email_aware_command_data_transformer" class="Sylius\Bundle\ApiBundle\DataTransformer\LoggedInShopUserEmailAwareCommandDataTransformer">
```
it can be fixed in a separate PR for all data transformers with adding an appropriate note in UPGRADE file

Reply:
Is needed not in UPGRADE file? Its new added in last PR without new tags: https://github.com/Sylius/Sylius/pull/12472/files#diff-32d98f7d054993ed0efa9e1348323ca84ff552c710dfa488b42fd6de6d494a1fR123